src/utils/disableNotification.util.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable require-jsdoc */
+import userService from '../services/authentication.service';
+import requestService from '../services/request.service';
+
+const { updateIsVerifiedOrDisableNotification, getUserById } = userService;
+const { updateInAppNotification } = requestService;
+const disableNotification = async (mode, id) => {

```

Comment:
Could you add empty lines before this function and after it, to separate it from the destructured statements above and the export statement below?